packages/cli/src/daemon-config.ts

```diff
@@ -1,6 +1,46 @@
 import 'reflect-metadata'
 import { jsonObject, jsonMember, jsonArrayMember, TypedJSON, toJson, AnyT } from 'typedjson'
 import { StreamID } from '@ceramicnetwork/streamid'
+import { readFile } from 'node:fs/promises'
+import { homedir } from 'os'
+
+const HOME_DIR = `${homedir()}/`.replace(/\/\/$/, '')
+
+/**
+ * Replace `~/` with `<homedir>/` absolute path.
+ * @param input Relative path.
+ */
+function expandHomedir(input: string): string {
```

Comment:
I'm really surprised we have to write this ourselves?  Is there really not a library that would handle this for us, in a platform agnostic way?

Reply:
if they do it bare on the command line the shell should do it for us, but if they quote it prob not - normally I think cli's don't expect to get '~' literals, the shell expands those?  but users might do it anyway.